library/src/main/java/com/getstream/sdk/chat/rest/core/Client.java

```diff
@@ -911,6 +913,78 @@ public void onError(String errMsg, int errCode) {
         });
     }
 
+    /**
+     * Query the API, get messages, members or other channel fields
+     *
+     * @param channel  query data for this channel
+     * @param request  request options
+     * @param callback the result callback
+     */
+    public void queryChannel(Channel channel, ChannelQueryRequest request, QueryChannelCallback callback) {
```

Comment:
Looks like a good move, the problem that it introduces new method to `Client` and we're trying to maintain the same API of Client for all platforms.
Let me check with Tommaso.

Reply:
Have you managed to discuss that?

Reply:
Didn't have chance. Let's keep this method for now. It's not breaking change, so shouldn't affect clients now.